### DISCUSSION TITLE
Roadmap in Readme is outdated.

### DISCUSSION THREAD

**Discussion by MaxwellDAssistek** [AvaloniaUI/Avalonia#13056] [General]
Hello,

I was reading through the GitHub Readme and I noticed that the Roadmap there is very out of date. It would be nice if it could be updated as we are very interested in where Avalonia is headed.

**Comment by thevortexcloud** [comment]
Someone recently asked a similar question which was answered by the Avalonia devs https://github.com/AvaloniaUI/Avalonia/discussions/12932

**Reply by MaxwellDAssistek** [reply]
Thanks. The suggestion still stands to either remove the link or make a new roadmap to avoid confusion as new people trickle in from the dumpster fire that is MAUI.

**Reply by timunie** [reply]
@MaxwellDAssistek your PR to remove the link for now is welcome imo. Good point. 

@MikeCodesDotNET
